main.py

```diff
@@ -122,11 +126,12 @@ def main():
                 syscalls = drakvufParser.parse_log(options.syscall, "syscall")
             else:
                 syscalls = drakvufParser.parse_log("syscall.log", "syscall")
+            detections = netParser.check(syscalls, 'drakvuf')
+            print_results('drakvuf', detections, netParser.net_num)
         elif "logtype" in source and source["logtype"] == "sysmon" and options.sysmon:
+            print()
```

Comment:
Is this supposed to be empty?

Reply:
I have moved the call to the `print_results()` function found in `main.py`.
```suggestion
            print('\n', end="")
```